projects/addon-charts/components/bar-set/bar-set.template.html

```diff
@@ -1,18 +1,14 @@
 <ng-container *ngIf="size; else flexible">
     <ng-container *ngFor="let item of computedValue; let index = index">
-        <ng-container
-            *ngTemplateOutlet="bar; context: {$implicit: item, index: index, flexible: false}"
-        ></ng-container>
+        <ng-container *ngTemplateOutlet="bar; context: {$implicit: item, index: index, flexible: false}"></ng-container>
```

Comment:
These could probably be changed to `[ngTemplateOutlet]` and `[ngTemplateOutletContext]` for improved readability and added to the `ng-container` with `ngFor` so there's less nesting.